java/core/src/test/java/co/worklytics/psoxy/storage/impl/BulkDataSanitizerImplTest.java

```diff
@@ -417,4 +417,54 @@ void pre_v0_4_30_bulk_pseudonym_URL_SAFE_TOKEN_ENCODING(String identifier) {
         assertEquals("t~" + legacyEncoded + (pseudonymizedIdentity.getDomain() == null ? "" : ("@" + pseudonymizedIdentity.getDomain())),
             urlSafeTokenPseudonymEncoder.encode(pseudonymizedIdentity.asPseudonym()));
     }
+
+
+    @SneakyThrows
+    @Test
+    void transform_ghusername() {
+
+        final String EXPECTED = "EMPLOYEE_ID,EMPLOYEE_EMAIL,DEPARTMENT,SNAPSHOT,MANAGER_ID,JOIN_DATE,LEAVE_DATE,GITHUB_USERNAME\r\n" +
+            "2,bob@workltyics.co,Sales,2023-01-06,1,2020-01-01,,\r\n" +
```

Comment:
I was trying to understand why bob and charles didn't get a GITHUB_USERNAME... subtle domain change `workltyics.co`...
and `workltycis.co`...
a comment would be welcome here.
Why are the domains messed up in the first place? wouldn't it be easier to just do `@subdomain.worklytics.co` or `@example.com` ??